web-frontend/src/pages/CreateAccount/StepTwo.tsx

```diff
@@ -0,0 +1,34 @@
+import { Stepper } from '@massalabs/react-ui-kit/src/components/Stepper/Stepper';
+import { Button } from '@massalabs/react-ui-kit/src/components/Button/Button';
+import { FiLock } from 'react-icons/fi';
+import LandingPage from '../../layouts/LandingPage/LandingPage';
+
+export default function StepTwo() {
+  return (
+    <LandingPage>
+      <div
+        className={`flex flex-col justify-center items-center align-center h-screen`}
+      >
+        <div className="flex flex-col justify-center items-center w-fit h-fit max-w-sm">
+          <div className=" w-full max-w-xs mb-6">
+            <Stepper step={1} steps={['Username', 'Password', 'Back-up']} />
```

Comment:
```suggestion
            <Stepper step={1} steps={['Username', 'Password', 'Backup']} />
```

Reply:
On Figma it's back-up, both are correct in English, which one should we write?